pkg/common-controller/snapshot_controller.go

```diff
@@ -95,6 +96,14 @@ func (ctrl *csiSnapshotCommonController) syncContent(content *crdv1.VolumeSnapsh
 		return err
 	}
 
+	// Perform additional validation. Label objects which fail.
+	// Part of a plan to tighten validation, this label will enable users to
+	// query for invalid content objects. See issue #363
+	if err := ctrl.checkAndAddInvalidContentLabel(content); err != nil {
```

Comment:
should i remove the previous check?

Reply:
I did not remove it. I moved the labeling right before the validation check that was there originally so that the original check wouldnt short circuit the labeling.